### PR TITLE
Generate image for store after uploading it

### LINKED ISSUE #32528: Multiple image formats : WebP is not generated when an image for a store is uploaded
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

WebP is not generated when a picture image for a store  is uploaded

### Expected behavior

WebP is generated when a picture image for a store  is uploaded

### Steps to reproduce

1. Enable Multiple image formats on the "New & Experimental Features" page
2. Enable WebP in Image Format page
3. Create a store with a JPEG/PNG file for picture file for thumb
4. The `img/st/{idStore}-stores_default.webp` doesn't exist => **ERROR**
5. When I go on the front office
6. The `img/st/{idStore}-stores_default.webp` exist

### PrestaShop version(s) where the bug happened

8.1.x

### PHP version(s) where the bug happened

7.3

### If your bug is related to a module, specify its name and its version

_No response_

### Your company or customer's name goes here (if applicable).

@PrestaShopCorp

### PR DESCRIPTION
**Pull request by Hlavtox**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.1.x
| Description?      | Generates all image formats for given store on the fly.
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | Follow steps in https://github.com/PrestaShop/PrestaShop/issues/32528.
| Fixed ticket?     | Fixes #32528
| Related PRs       | 
| Sponsor company   | 

Ping @Progi1984 - can you test please? :-)

### CODE REVIEW

**Comment by Progi1984** [comment]
@Hlavtox Uncomment these lines in your PR : https://github.com/PrestaShop/PrestaShop/blob/8.1.x/tests/UI/campaigns/functional/BO/08_design/06_imageSettings/15_checkStoreImageFormat.ts#L221-L230

And after, you can test using `PrestaShop/ga.tests.ui.pr`

File: controllers/admin/AdminStoresController.php [hunk 1]
```diff
@@ -415,23 +419,45 @@ protected function postImage($id)
         // Should we generate high DPI images?
         $generate_hight_dpi_images = (bool) Configuration::get('PS_HIGHT_DPI');
```
Comment (SharakPL) [inline controllers/admin/AdminStoresController.php:420]: Naming here is confusing. But it probably should be fixed in another PR...


```suggestion
         $generate_high_dpi_images = (bool) Configuration::get('PS_HIGH_DPI');
```

**Review by SharakPL — DISMISSED** [review]

File: controllers/admin/AdminStoresController.php (hunk 1 above)
Comment (Hlavtox) [reply controllers/admin/AdminStoresController.php:420]: @SharakPL I removed this feature from v9 completely, so it's fixed. 😁

**Review by boherm — DISMISSED** [review]

**Review by Progi1984 — CHANGES_REQUESTED** [review]
@Hlavtox Uncomment these lines in your PR : https://github.com/PrestaShop/PrestaShop/blob/8.1.x/tests/UI/campaigns/functional/BO/08_design/06_imageSettings/15_checkStoreImageFormat.ts#L221-L230

And after, you can test using `PrestaShop/ga.tests.ui.pr`

**Comment by Hlavtox** [comment]
Test run: https://github.com/Hlavtox/ga.tests.ui.pr/actions/runs/5555440205

**Review by Progi1984 — APPROVED** [review]
Ok for UI tests

**Review by boubkerbribri — APPROVED** [review]

**Review by kpodemski — APPROVED** [review]

**Review by boherm — APPROVED** [review]

**Review by aniszr — APPROVED** [review]
Hello @Hlavtox 

I tested your PR and LGTM :heavy_check_mark: 

Auto Test :heavy_check_mark: except the [FacetedSearch](https://github.com/aniszr/ga.tests.ui.pr/actions/runs/5608916573/jobs/10264058327) module uninstall and I believe it caused by this bug https://github.com/PrestaShop/PrestaShop/issues/33295
PS: @Hlavtox could you please confirm that it's not related to your PR :pray: 
See attached screenrecord :

https://github.com/PrestaShop/PrestaShop/assets/92626242/f1c2a19b-0736-46a0-9e6b-f70945767a0a

Thanks!

**Comment by Progi1984** [comment]
@aniszr I confirm it's not related.

**Pull request merged**